src/Blt/Plugin/Commands/CohesionCommands.php

```diff
@@ -29,25 +35,86 @@ public function postCommand($result, CommandData $commandData)
-        ->run();
+      if (!isset($sync_import) || $sync_import == TRUE) {
+        // Import Site Studio configuration from the sync folder.
+        $result = $this->taskDrush()
+          ->stopOnFail()
+          ->drush("sync:import --overwrite-all --force")
+          ->run();
+      } else {
+        $this->say("Cohesion Sync Import disabled via blt.yml, skipping.");
+      }
 
-      // Rebuild Site Studio.
-      $result = $this->taskDrush()
-        ->stopOnFail()
-        ->drush("cohesion:rebuild")
-        ->run();
+      if (!isset($rebuild) || $rebuild == TRUE) {
+        // Rebuild Site Studio.
+        $result = $this->taskDrush()
+          ->stopOnFail()
+          ->drush("cohesion:rebuild")
+          ->run();
+      } else {
+        $this->say("Cohesion Rebuild disabled via blt.yml, skipping.");
+      }
     } else {
       $this->say("Site Studio sync is not enabled. Skipping Site Studio import and rebuild.");
     }
   }
+
+  /**
+   * Initializes default Site Studio config split configuration for this project.
+   *
+   * @command recipes:config:init:sitestudio
+   * @throws \Acquia\Blt\Robo\Exceptions\BltException
+   */
+  public function generateSiteStudioConifg() {
+    $split_dir = $this->getConfigValue('repo.root') ."/config/site_studio_sync";
+    $this->say("This command will automatically generate and place configuration and settings files for Site Studio.");
+    $result = $this->taskFilesystemStack()
+      ->mkdir($split_dir)
+      ->run();
+    if (!$result->wasSuccessful()) {
+      throw new BltException("Unable to create $split_dir.");
+    }
+    $result = $this->taskFilesystemStack()
+      ->copy($this->getConfigValue('repo.root') . '/vendor/acquia/blt-site-studio/config/config_split.config_split.site_studio_ignored_config.yml', $this->getConfigValue('repo.root') . '/config/default/config_split.config_split.site_studio_ignored_config.yml', TRUE)
+      ->copy($this->getConfigValue('repo.root') . '/vendor/acquia/blt-site-studio/config/config_ignore.settings.yml', $this->getConfigValue('repo.root') . '/config/default/config_ignore.settings.yml', TRUE)
+      ->copy($this->getConfigValue('repo.root') . '/vendor/acquia/blt-site-studio/config/.htaccess', $this->getConfigValue('repo.root') . '/config/site_studio_sync/.htaccess', TRUE)
+      ->copy($this->getConfigValue('repo.root') . '/vendor/acquia/blt-site-studio/config/README.md', $this->getConfigValue('repo.root') . '/config/site_studio_sync/README.md', TRUE)
+      ->copy($this->getConfigValue('repo.root') . '/vendor/acquia/blt-site-studio/settings/includes.settings.php', $this->getConfigValue('docroot') . 'sites/default/config/default/settings/includes.settings.php', TRUE)
+      ->copy($this->getConfigValue('repo.root') . '/vendor/acquia/blt-site-studio/settings/site-studio.settings.php', $this->getConfigValue('docroot') . 'sites/default/config/default/settings/site-studio.settings.php', TRUE)
```

Comment:
`sites/default/config/default/settings` - I think the `config/default` needs to be removed.